x/ibc/07-tendermint/types/misbehaviour.go

```diff
@@ -103,6 +105,17 @@ func (cs ClientState) CheckMisbehaviourAndUpdateState(
 func checkMisbehaviourHeader(
```

Comment:
for some reason this function is not tested?

Reply:
since this is an existing issue on master I think it is outside the scope of this pr. @AdityaSripal could you pick this up :pray: